compiler/src/language_server/hover.re

```diff
@@ -126,11 +126,17 @@ let declaration_lens = (ident: Ident.t, decl: Types.type_declaration) => {
 };
 
 let include_lens = (env: Env.t, path: Path.t) => {
-  let module_decl = Env.find_module(path, None, env);
-  markdown_join(
-    grain_code_block("module " ++ Path.name(path)),
-    module_lens(module_decl),
-  );
+  let header = grain_code_block("module " ++ Path.name(path));
+  let decl = Env.find_module(path, None, env);
+  let module_decl =
+    switch (Modules.get_provides(decl)) {
+    | [_, ..._] => Some(module_lens(decl))
+    | [] => None
+    };
+  switch (module_decl) {
+  | Some(mod_sig) => markdown_join(header, mod_sig)
+  | None => header
```

Comment:
If there's no module declaration should we be showing the header?

Reply:
I think so. I think of a module that exists but doesn't have any exports as empty.